lightning/src/events/mod.rs

```diff
@@ -1582,6 +1582,55 @@ pub enum Event {
 		/// onion messages.
 		peer_node_id: PublicKey,
 	},
+	/// Indicates that a funding transaction constructed via interactive transaction construction for a
+	/// channel is ready to be signed by the client. This event will only be triggered
+	/// if at least one input was contributed by the holder and needs to be signed.
```

Comment:
Not sure if we should we expose "holder" to the user. We might do that elsewhere, but I'm not sure if readers of these docs will understand what that means. Instead, we may want to reference the method that is called to contribute inputs.